BioCro/DARPA/upload_params.Rmd

```diff
@@ -147,30 +147,41 @@ done
 ```
 
 
-## Specific leaf area
+## Specific Leaf Area
 
-These SLA values are only for the high night temperature treatment from experiment 3rd_Biomass_ME034_GA_Exp. Read in, clean up, and combine leaf biomass measurements with corresponding leaf area measurements. 
+These SLA values are calculated from leaf area and leaf dry biomass. Below, the data are read in, filtered, and combined to produce specific leaf area in units of m2/kg. 
 
 These are the final columns:
 
-1. `local_datetime`: convert date into machine readable format
-2. `treatment`: specifies record's treatment using BETYdb treatment names
-3. `SLA`: measured values of specific leaf area
+
+1. `citation_author`, `citation_year`, and `citation_title`: indicate the source of the data using author, year, and title from BETYdb
+2. `species`: Setaria viridis, `cultivar`: ME-034
+3. `site`: one of three sites at Donald Danforth Plant Science Center, either Growth Chamber, Greenhouse, or Outdoor, using BETYdb site names
+4. `treatment`: specifies record's treatment using BETYdb treatment names
+5. `local_datetime`: convert date into machine readable format
+6.  `SLA`: Specific Leaf Area calculated from single-sided fresh leaf area divided by leaf dry mass, m2/kg
+8. `n`: sample size, always set to 1
+9. `access_level`: 2, equivalent to Internal & Collaborators
+
+
+First, load in data from the chamber experiments, an Excel sheet entitled "manual-measurements-Darpa_setaria_chambers_experiments.xlsx", which consists of multiple tabs. All leaf areas are avaiable in tab 'total_leaf_area'; biomass for the high night temperature experiment are found in '3rd_Biomass_ME034_GA_Exp' and the biomass for the high light experiment are found in '5th_Biomass_A10&ME034_cycling_t'. Note that because physiological and biomass data were taken on different individual plants, unique filters are applied to each biomass data to obtain the same set of experimental conditions as the physiological data. 
 ```{r}
-data_path <- "../../../model-vignettes-data/manual-measurements-Darpa_setaria_chambers_experiments.xlsx"
+# Data from chamber experiments
+data_path <- "../../../sentinel-detection/data/raw_data/biomass/manual-measurements-Darpa_setaria_chambers_experiments.xlsx"
 sheets_names <- excel_sheets(data_path)
 
+# Biomass and leaf area for high light treatment, 31_31_250
 leaf_biomass_hightemp <- read_excel(data_path, sheets_names[11]) %>% 
```

Comment:
My instincts are telling me that there's enough redundancy across treatments/conditions in the cleaning and combining of the biomass and leaf area data that they should be refactored into one function. But each of these seems to be just different enough that that doesn't make sense to do? I'm not entirely sure, it's up to you @jessicaguo.

Reply:
You are correct on all counts Should be more streamlined, but there are just enough differences in the data sheets that makes that difficult. I found and requested changes to the raw data sheet several times just to get it to this stage!